testapi.pm

```diff
@@ -294,6 +294,10 @@ sub wait_idle {
     my $timeout = shift || $bmwqemu::idle_timeout;
     $timeout = bmwqemu::scale_timeout($timeout);
 
+    # report wait_idle calls more offensiv to help with https://progress.opensuse.org/issues/5830
+    use Carp qw(cluck);
+    cluck "Wait_idle called";
```

Comment:
IMHO it's deprecated but not obsolete. "deprecated" means "please don't use" as in "reduce it's usage". That's what you said.